libs/json-api/json-shared-type/src/types/entity-type.ts

```diff
@@ -4,6 +4,7 @@ export type EntityField =
   | boolean
   | string[]
   | number[]
+  | null
   | Date;
```

Comment:
adding `bigint` as well or not necessary? 
It is already available on psql (numeric) and typescript (bigint) as well